packages/forma-36-react-components/src/components/Button/Button.css

```diff
@@ -29,48 +25,58 @@
       var(--transition-easing-default);
   vertical-align: middle;
   text-decoration: none;
+  box-shadow: 0px 1px 0px rgb(25 37 50 / 10%);
 
   &:hover:not(.Button--disabled) {
     cursor: pointer;
   }
+
+  &:focus:not(.Button--disabled) {
+    outline: none;
+  }
 }
 
 .Button__icon {
   min-width: calc(1rem * (18 / var(--font-base-default)));
+  margin-right: calc(1rem * (8 / var(--font-base-default)));
+}
+
+.Button__dropdown-icon {
+  margin-left: calc(1rem * (8 / var(--font-base-default)));
 }
 
 .Button--full-width {
   width: 100%;
 }
 
 .Button--muted {
-  background-color: var(--color-element-light);
+  background-color: var(--color-white);
+  border-color: var(--color-element-dark);
+  box-shadow: 0px 1px 0px rgb(25 37 50 / 4%);
 
-  &:hover:not(.Button--disabled),
-  &:focus:not(.Button--disabled) {
-    background-color: var(--color-element-mid);
+  &.Button--is-active:not(.Button--disabled),
+  &:active:not(.Button--disabled),
+  &:hover:not(.Button--disabled) {
+    background-color: var(--color-element-lightest);
   }
 
-  &:active:not(.Button--disabled),
-  &.Button--is-active:not(.Button--disabled) {
-    transition: none;
-    background-color: var(--color-element-dark);
+  &:focus:not(.Button--disabled) {
+    box-shadow: 0px 0px 0px 3px var(--color-element-light);
```

Comment:
@gui-santos so here it would be possible to use glow from tokens?

Reply:
Oops! I forgot to replace the values
I will do that now